ThePortugueseMan.FlashCards/Screens/StudySessionMenu.cs

```diff
@@ -0,0 +1,186 @@
+﻿using AskInputs;
+using ConsoleTableExt;
+using ObjectsLibrary;
+
+namespace Screens;
+
+internal class StudySessionMenu
+{
+    AskInput askInput = new AskInput();
+    SettingsLibrary.Settings settings = new();
+    DbCommandsLibrary.DbCommands dbCmd = new();
+    public void View()
+    {
+        bool exitMenu = false;
+        List<object> optionsString = new List<object> {
+            "1 - New Session",
+            "2 - See Sessions",
+            "0 - Return"
+        };
+        while (!exitMenu)
+        {
+            Console.Clear();
+            ConsoleTableBuilder.From(optionsString)
+                .WithFormat(ConsoleTableBuilderFormat.Alternative)
+                .WithColumn("Study Sessions")
+                .ExportAndWriteLine();
+            Console.Write("\n");
+
+            switch (askInput.PositiveNumber("Please select a valid option"))
+            {
+                case 0: exitMenu = true; continue;
+                case 1:  NewStudyMenu(); break;
+                case 2:  ViewAllSessions(); break;
+                default: break;
+            }
+        }
+        return;
+    }
+
+    public void NewStudyMenu()
+    {
+        Console.Clear();
+        int stackViewId, stackIndex;
+        StudySession studySession = new();
+        Stack stack = new();
+
+        StacksMenu stacks = new();
+        stacks.DisplayStackList(dbCmd.Return.AllStacks(), "CHOOSE");
+
+        do
+        {
+            stackViewId = askInput.PositiveNumber("Choose the stack you want to study, or 0 to return");
```

Comment:
Hey! Project is looking great! Just a bit unsure what is the behavior expected in the study session area? As an user it wasn't clear what the functionality is. I think adding a bit more explanation will do? 

![image](https://user-images.githubusercontent.com/108711476/226766569-11f9d29b-d027-4c70-ba31-c719206351d2.png)

Reply:
Hey, thanks for the feedback! I've updated the code.

![image](https://user-images.githubusercontent.com/64802476/227206262-d1f60aec-0ff8-45d7-993d-20e29fd2b69c.png)


The user was supposed to press the left/right arrow if the got the answer wrong/right. Similar to how a traditional flashcard system would work.

I hope that's clear, and thanks for the review!